docs/src/content/docs/guides/recurring-remittance-fixed-receive.mdx

```diff
@@ -0,0 +1,538 @@
+
+- `receiveAmount` - The maximum amount that the recipient can receive per interval. When the next interval begins, the value resets.
+- `interval` - The [time interval](#about-the-interval) under which the grant is valid.
+
+:::note
+Outgoing payments require an interactive grant, which the sender approves once. While the grant (and its access token) is valid and within its limits (`interval` + `receiveAmount`), your app can create outgoing payments without re-approval each interval. You can find more information in the [Open Payments flow](/concepts/op-flow/#outgoing-payment) and [identity providers](/identity/idp) pages.
+:::
+
+<Tabs syncKey="lang">
+<TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const pendingSenderOutgoingPaymentGrant = await client.grant.request(
+    {
+      url: senderWalletAddress.authServer
+    },
+    {
+      access_token: {
+        access: [
+          {
+            identifier: senderWalletAddress.id,
+            type: 'outgoing-payment',
+            actions: ['create'],
+            limits: {
+              interval: 'R3/2025-10-03T23:25:00Z/P1M',
+              receiveAmount: {
+                assetCode: 'MXN',
+                assetScale: 2,
+                value: '400000', 
+              }
+            }
+          }
+        ]
+      },
+      interact: {
+        start: ['redirect'],
+        finish: {
+          method: 'redirect',
+          uri: 'https://myapp.example.com/finish/{...}', // where to redirect your user after they've completed the interaction
+          nonce: NONCE
+        }
+      }
+    }
+  )
+  ```
+</TabItem>
+</Tabs>
+
+<details>
+<summary>Example response</summary>
+The following shows an example response from the sender's wallet provider.
+```json wrap
+{
+  "interact": {
+    "redirect": "https://auth.interledger-test.dev/{...}", // uri to redirect your user to, to begin interaction
+    "finish": "..." // unique key to secure the callback
+  },
+  "continue": {
+    "access_token": {
+      "value": "..." // access token for continuing the outgoing payment grant request
+    },
+    "uri": "https://auth.interledger-test.dev/continue/{...}", // uri for continuing the outgoing payment grant request
+    "wait": 30
+  }
+}
+```
+</details>
+
+#### About the interval
+
+The interval used in this guide is `R3/2025-10-03T23:25:00Z/P1M`. Remember that the sender wants the recipient to receive $4,000 MXN a month for three months. The interval breaks down like this:
+
+- `R3/` is the number of repetitions - three
+- `2025-10-03` is the start date of the repeating interval - 03 October 2025
+- `T23:25:00Z/` is the start time of the repeating interval - 11:25 PM UTC
+- `P1M` is the period between each interval - one month. Used with `R3`, you have a grant that's valid once a month for three months.
+
+Altogether, this grant will allow the sender to make outgoing payments from:
```

Comment:
```suggestion
Altogether, this grant will allow the sender to make any number of outgoing payments within the defined limit from:
```

Reply:
Thanks for this. It was something that dawned on me as I was working through it, but didn't go back. We will fix that in other guides as well.